trace_viewer/base/ui/mouse_mode_selector.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <!--
-Copyright (c) 2013 The Chromium Authors. All rights reserved.
-Use of this source code is governed by a BSD-style license that can be
-found in the LICENSE file.
+ Copyright (c) 2015 The Chromium Authors. All rights reserved.
```

Comment:
nit: Remove the spaces added at the start of the block.

We don't, typically, update the copyright year when editing files.